GDevelop.js/types/gdvariablepointer.js

```diff
@@ -0,0 +1,6 @@
+// Automatically generated by GDevelop.js/scripts/generate-types.js
+declare class gdVariablePointer {
```

Comment:
Nitpicking: let's call this a `SharedPtrVariable` (much like we have `UniquePtrExpressionNode`).

BUT let's check if we even need this class. I'm not sure, because in 99% of cases we don't need to expose the shared pointers to the JavaScript - it's an implementation detail and we can use `GetAt()`/`at()` to get reference to a variable (no shared pointer) and `GetChildrenCount()`/`size()` to get the count.

Reply:
When I had put only `[Const, Ref] Variable at();`, it made a compilation error that said that it couldn't convert a pointer to a smart pointer.

Reply:
Ah yes didn't see there was a wrapped at that calls get automatically. Trying it out right now.